shipengine_sdk/models/package/__init__.py

```diff
@@ -26,8 +26,9 @@ def __init__(
     ) -> None:
         """This object represents a given Shipment."""
         self.config = config
-        self.shipment_id = shipment["shipmentID"] if "shipmentID" in shipment else None
-        self.account_id = shipment["carrierAccountID"] if "carrierAccountID" in shipment else None
+        self.shipment_id = shipment["shipmentId"] if "shipmentId" in shipment \
```

Comment:
You only want to use the `\` when the line is too long (120 chars) and if you run `poetry run black -v shipengine_sdk` it should reformat that and fix this for you automatically. It runs as part of pre-commit and I know we don't have docs on pre-commit on this branch so it's not a huge deal.